lib/manageiq/api/common/rbac/unshare_resource.rb

```diff
@@ -0,0 +1,30 @@
+require 'rbac-api-client'
+module ManageIQ
+  module API
+    module Common
+      module RBAC
+        class UnshareResource < ShareResource
```

Comment:
What is an unshare resource? and how can an unshare be a subclass of a share?

Reply:
The `share` and `unshare` resource are different, but they share the similar codes.